The build failed

This test failed!

To configure my behavior, see [the Flaky Bot documentation](https://github.com/googleapis/repo-automation-bots/tree/main/packages/flakybot).

If I'm commenting on this issue too often, add the `flakybot: quiet` label and
I will stop commenting.

---

commit: 2090e822906f899d9bee9a9cffdf96f8b707e925
buildURL: [Build Status](https://source.cloud.google.com/results/invocations/a8641e3b-d95e-48d0-8dbc-8c21376a10cd), [Sponge](http://sponge2/a8641e3b-d95e-48d0-8dbc-8c21376a10cd)
status: failed
<details><summary>Test output</summary><br><pre>java.lang.AssertionError: 
Dataset was not deleted. 
com.google.cloud.bigquery.BigQueryException: Dataset java-docs-samples-testing:sampledataset is still in use
	at BigQueryExportIT.deleteBigQueryDataset(BigQueryExportIT.java:120)
	at BigQueryExportIT.cleanUp(BigQueryExportIT.java:88)
</pre></details>

```
#
# media/livestream
#

[0.001s][warning][os,container] Duplicate cpuset controllers detected. Picking /sys/fs/cgroup/cpuset, skipping /tmpfs/docker/overlay2/f43673db59b20a61f8c250e5316cb9c4098f6532849ce4b388f4e082f79a807b/merged/sys/fs/cgroup/cpuset.
[ERROR] Tests run: 2, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 6.782 s <<< FAILURE! - in com.example.livestream.StartChannelTest
[ERROR] com.example.livestream.StartChannelTest.test_StartChannel  Time elapsed: 6.774 s  <<< ERROR!
java.util.concurrent.ExecutionException: com.google.api.gax.rpc.UnavailableException: io.grpc.StatusRuntimeException: UNAVAILABLE: 502:Bad Gateway

[ERROR] com.example.livestream.StartChannelTest.test_StartChannel  Time elapsed: 6.778 s  <<< ERROR!
java.util.concurrent.ExecutionException: com.google.api.gax.rpc.NotFoundException: io.grpc.StatusRuntimeException: NOT_FOUND: Resource 'projects/java-docs-samples-testing/locations/us-central1/channels/my-channel-54abf171-61fb-49fb-9b6e-b' was not found

[ERROR] Errors:
[ERROR] com.example.livestream.StartChannelTest.test_StartChannel
[ERROR]   Run 1: StartChannelTest.beforeTest:87 » Execution com.google.api.gax.rpc.UnavailableE...
[ERROR]   Run 2: StartChannelTest.tearDown:104 » Execution com.google.api.gax.rpc.NotFoundExcep...
[ERROR] Tests run: 17, Failures: 0, Errors: 1, Skipped: 0

#
# security-command-center/snippets
#

[0.001s][warning][os,container] Duplicate cpuset controllers detected. Picking /sys/fs/cgroup/cpuset, skipping /tmpfs/docker/overlay2/f43673db59b20a61f8c250e5316cb9c4098f6532849ce4b388f4e082f79a807b/merged/sys/fs/cgroup/cpuset.
[ERROR] Tests run: 4, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 21.331 s <<< FAILURE! - in BigQueryExportIT
[ERROR] BigQueryExportIT  Time elapsed: 3.74 s  <<< FAILURE!
java.lang.AssertionError:
Dataset was not deleted.
com.google.cloud.bigquery.BigQueryException: Dataset java-docs-samples-testing:sampledataset is still in use
	at BigQueryExportIT.deleteBigQueryDataset(BigQueryExportIT.java:120)
	at BigQueryExportIT.cleanUp(BigQueryExportIT.java:88)

[ERROR] Failures:
[ERROR]   BigQueryExportIT.cleanUp:88->deleteBigQueryDataset:120 Dataset was not deleted.
com.google.cloud.bigquery.BigQueryException: Dataset java-docs-samples-testing:sampledataset is still in use
[ERROR] Tests run: 9, Failures: 1, Errors: 0, Skipped: 0



#
# Summary
#

SUCCESS: 259, FAILURE: 2, SKIPPED: 0, ERROR: 0, 
media/livestream......................................................[ FAILURE]
security-command-center/snippets......................................[ FAILURE] 
```

--- a/security-command-center/snippets/src/test/java/BigQueryExportIT.java
+++ b/security-command-center/snippets/src/test/java/BigQueryExportIT.java
@@ -27,6 +27,7 @@ import com.google.cloud.bigquery.BigQueryException;
 import com.google.cloud.bigquery.BigQueryOptions;
 import com.google.cloud.bigquery.Dataset;
 import com.google.cloud.bigquery.DatasetInfo;
+import com.google.cloud.testing.junit4.MultipleAttemptsRule;
 import java.io.ByteArrayOutputStream;
 import java.io.IOException;
 import java.io.PrintStream;
@@ -36,16 +37,19 @@ import org.junit.AfterClass;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.BeforeClass;
+import org.junit.Rule;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
 @RunWith(JUnit4.class)
 public class BigQueryExportIT {
+  @Rule public final MultipleAttemptsRule multipleAttemptsRule = new MultipleAttemptsRule(5);
 
   // TODO(Developer): Replace the below variables.
   private static final String PROJECT_ID = System.getenv("GOOGLE_CLOUD_PROJECT");
-  private static final String BQ_DATASET_NAME = "sampledataset";
+  private static final String BQ_DATASET_NAME =
+      "sampledataset-" + UUID.randomUUID().toString().split("-")[0];
   private static final String BQ_EXPORT_ID =
       "default-" + UUID.randomUUID().toString().split("-")[0];
 

--- a/security-command-center/snippets/src/test/java/BigQueryExportIT.java
+++ b/security-command-center/snippets/src/test/java/BigQueryExportIT.java
@@ -49,7 +49,7 @@ public class BigQueryExportIT {
   // TODO(Developer): Replace the below variables.
   private static final String PROJECT_ID = System.getenv("GOOGLE_CLOUD_PROJECT");
   private static final String BQ_DATASET_NAME =
-      "sampledataset-" + UUID.randomUUID().toString().split("_")[0];
+      "sampledataset_" + UUID.randomUUID().toString().split("-")[0];
   private static final String BQ_EXPORT_ID =
       "default-" + UUID.randomUUID().toString().split("-")[0];
 

--- a/security-command-center/snippets/src/test/java/BigQueryExportIT.java
+++ b/security-command-center/snippets/src/test/java/BigQueryExportIT.java
@@ -49,7 +49,7 @@ public class BigQueryExportIT {
   // TODO(Developer): Replace the below variables.
   private static final String PROJECT_ID = System.getenv("GOOGLE_CLOUD_PROJECT");
   private static final String BQ_DATASET_NAME =
-      "sampledataset-" + UUID.randomUUID().toString().split("-")[0];
+      "sampledataset-" + UUID.randomUUID().toString().split("_")[0];
   private static final String BQ_EXPORT_ID =
       "default-" + UUID.randomUUID().toString().split("-")[0];
 

